static/src/javascripts/projects/common/modules/onward/tech-feedback.js

```diff
@@ -37,6 +40,18 @@ const getExtraDataInformation = (): Object => ({
     abTests: summariseAbTests(getABParticipations()),
 });
 
+const getExtraDataInformation = (): Promise<Object> =>
```

Comment:
I think this should already have been stored in a cookie in [user-features](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/commercial/user-features.js)